be/src/exec/local_file_reader.cpp

```diff
@@ -36,53 +36,71 @@ Status LocalFileReader::open() {
             << ", error=" << strerror_r(errno, err_buf, 64);
         return Status(ss.str());
     }
+    // get file size
```

Comment:
you'd better not get size in open function

you can get through `size` function

Reply:
i changed